README.md

```diff
@@ -100,7 +109,7 @@ While Mint supports authentication via Voice, `mintapi` does not currently suppo
 
 ### Multi-Data Support
 
-As of v2.0, MintAPI supports returning multiple types of data in one call to MintAPI.  When exporting multiple data types, you can either send it directly to `stdout` or you can export to a data file.  What MintAPI will do with your specified filename is add a suffix based on the type of data you are exporting.  The following table outlines the option selected and its corresponding suffix:
```

Comment:
I changed these to `mintapi` only because the rest of the README predominantly uses this formatting and wanted it to be consistent.